<ui version="4.0" >
 <class>addsoftnamebase</class>
 <widget class="QDialog" name="addsoftnamebase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add software name</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_softno" >
     <property name="text" >
      <string>Software no</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="lineEdit_softno" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_softname" >
     <property name="text" >
      <string>Software name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="lineEdit_softname" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_softdec" >
     <property name="text" >
      <string>Software description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QTextEdit" name="textEdit_softdec" />
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
